components/map/MapZoneBottomSheetAttachment.tsx

```diff
@@ -30,23 +33,30 @@ const MapZoneBottomSheetAttachment = ({ setFlyToCenter, ...restProps }: Props) =
     }
   }, [setFlyToCenter, permissionStatus])
 
+  const { tickets } = useTickets(true)
+  const activeTicketsCount = tickets?.length ?? 0
+
   return (
     <BottomSheetTopAttachment {...restProps}>
       <FlexRow cn="flex-1 p-2.5 pt-0 items-end">
-        <View className="rounded-full bg-white shadow ">
-          <Link asChild href="/tickets">
-            <PressableStyled>
-              <FlexRow cn="g-2 p-2 pr-3 items-center">
-                <View className="h-8 w-8 items-center justify-center rounded-full bg-light">
-                  <Icon size={20} name="local-parking" />
-                </View>
-                <Typography variant="default-bold" className="leading-6">
-                  Tickets (2)
-                </Typography>
-              </FlexRow>
-            </PressableStyled>
-          </Link>
-        </View>
+        {activeTicketsCount > 0 ? (
```

Comment:
Our practice is usually to check it like this. You have to use ternary in this case always - `undefined`, `null` or `0` evaluates to false, that is what we want usually:
```
tickets?.length ? <Component/> : null
```

And you don't have to create unnecessary variable :)